sdk/appconfiguration/app-configuration/package.json

```diff
@@ -69,6 +69,10 @@
       {
         "path": "src/generated/src/appConfigurationContext.ts",
```

Comment:
While we're editing these files for the purpose of keeping versions up to date, it looks like `packageVersion` has been removed from `appConfigurationContext.ts` so we can remove this item from the list.